tests/runtime/basic

```diff
@@ -145,3 +145,13 @@ NAME tuple print
 RUN bpftrace -e 'BEGIN { @ = (1, 2, "string", (4, 5)); exit(); }'
 EXPECT ^@: \(1, 2, string, \(4, 5\)\)$
 TIMEOUT 5
+
+NAME disable warning: without --no-warnings
+RUN bpftrace  -e 'BEGIN { @x = stats(10); print(@x, 2); clear(@x); exit();}' 2>&1| grep -c  "WARNING"
+EXPECT ^1$
+TIMEOUT 1
```

Comment:
We should already have tests for warning output. Runtime tests are relatively expensive b/c of how long they take to run. Not sure we should have a duplicated test.